tests/phpunit/api/v3/ContributionPageTest.php

```diff
@@ -1795,26 +1793,26 @@ public function testSubmitPledgePaymentPaymentProcessorRecurFuturePayment() {
 
     // Check if pledge created.
     $pledge = $this->callAPISuccess('pledge', 'getsingle', []);
-    $this->assertEquals(date('Ymd', strtotime($pledge['pledge_start_date'])), date('Ymd', strtotime("+1 month")));
-    $this->assertEquals($pledge['pledge_amount'], 300.00);
+    $this->assertEquals(date('Ymd', strtotime($pledge['pledge_start_date'])), date('Ymd', strtotime('+1 month')));
```

Comment:
The configured value just happens to be 30 rather than 300 - change tests to match